mllib/src/main/scala/org/apache/spark/ml/clustering/LDA.scala

```diff
@@ -224,6 +224,20 @@ private[clustering] trait LDAParams extends Params with HasFeaturesCol with HasM
   /**
    * For Online optimizer only: [[optimizer]] = "online".
    *
+   * @group expertParam
+   */
+  @Since("2.3.0")
+  final val epsilon = new DoubleParam(this, "epsilon", "(For online optimizer)" +
+    " A (positive) learning parameter that controls the convergence of variational inference.",
```

Comment:
The parameter introduction here cannot really help a user without knowledge of LDA implementation. Please add more description for the effect if user want to tune the parameter, such like "Smaller value will lead to higher accuracy with the cost of more iterations."